tests/integration/test_tls.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+import pytest as pytest
+from pytest_operator.plugin import OpsTest
+
+from tests.integration.helpers.helpers import (
+    PGB,
+    PGB_METADATA,
+    POSTGRESQL_APP_NAME,
+    deploy_and_relate_application_with_pgbouncer,
+)
+
+MATTERMOST_APP_NAME = "mattermost"
```

Comment:
Just a minor, I think app name mattermost is now symlink to mattermost-k8s, 
try to open https://charmhub.io/mattermost

Thi sis why it works here, while the real app name is now mattermost-k8s, isn't it?

It will also allow you to use MATTERMOST_APP_NAME in like 54.

Reply:
That's app name within juju. What is deployed is mattermost-k8s charm.